modules/distr-tracing-architecture.adoc

```diff
@@ -25,6 +25,21 @@ This module included in the following assemblies:
 
 ** *Jaeger Console* – With the {JaegerName} user interface, you can visualize your distributed tracing data. On the Search page, you can find traces and explore details of the spans that make up an individual trace.
 
+* *{TempoName}* - This component is based on the open source link:https://github.com/grafana/tempo/[Grafana Tempo project].
+
+** *Gateway* – The Gateway handles authentication, authorization and forwards requests to the Distributor or Query frontend service.
+
+** *Distributor* – The Distributor accepts spans in multiple formats including Jaeger, OpenTelemetry, Zipkin. It routes spans to ingesters by hashing the `+traceID+` and using a distributed consistent hash ring.
+
+** *Ingester* – The Ingester batches trace into blocks, creates bloom filters and indexes, and then flushes it all to the backend. Blocks in the backend are generated in the following layout.
+
+** *Query Frontend* – The Query Frontend is responsible for sharding the search space for an incoming query. The search query is then sent to the Queriers. The Query Frontend deployment exposes Jaeger UI through Tempo Query sidecar.
+
+** *Querier* - The Querier is responsible for finding the requested trace id in either the ingesters or the backend storage. Depending on parameters it will query both the ingesters and pull bloom/indexes from the backend to search blocks in object storage.
```

Comment:
```suggestion
** *Querier* - The Querier is responsible for finding the requested trace ID in either the Ingesters or the back-end storage. Depending on parameters, it can query the Ingesters and pull Bloom indexes from the back end to search blocks in object storage.
```

Reply:
It first searches blocks from the storage and then once they are loaded in the memory it searches through the blocks to find the data

Reply:
Thank you for the explanation.